paasta_tools/docker_wrapper.py

```diff
@@ -166,6 +169,37 @@ def add_argument(args, argument):
     return args
 
 
+def overwrite_argument(args: List[str], keys: List[str], new_value: str) -> List[str]:
```

Comment:
could we do this overwrite before we get to the point that we're working with docker args? we should ideally just be giving the docker wrapper an already pre-processed set of resource requirements